File: Languages/Clojure/normal.clj
(ns normal.clj)

(defn adicionar-tarefa [lista tarefa]
  (conj lista tarefa))

(defn remover-tarefa [lista tarefa]
  (remove #(= % tarefa) lista))

(defn listar-tarefas [lista]
  (println "Tarefas:")
  (doseq [t lista]
    (println "- " t)))

(defn menu []
  (let [tarefas (atom [])]
    (loop []
      (println "\nMenu:")
      (println "1. Adicionar tarefa")
      (println "2. Remover tarefa")
      (println "3. Listar tarefas")
      (println "4. Sair")
      (let [opcao (read-line)]
        (cond
          (= opcao "1") (do
                          (println "Digite a tarefa:")
                          (let [tarefa (read-line)]
                            (swap! tarefas adicionar-tarefa tarefa))
                          (recur))

          (= opcao "2") (do
                          (println "Digite a tarefa a ser removida:")
                          (let [tarefa (read-line)]
                            (swap! tarefas remover-tarefa tarefa))
                          (recur))

          (= opcao "3") (do
                          (listar-tarefas @tarefas)
                          (recur))

          (= opcao "4") (println "Saindo...")

          :else (do
                  (println "Opção inválida.")
                  (recur)))))))

(defn -main []
  (menu))

File: Languages/Python/normal.py
def display_tasks(tasks):
    if not tasks:
        print("Nenhuma tarefa na lista.")
    else:
        print("Tarefas:")
        for index, task in enumerate(tasks, start=1):
            print(f"{index}. {task}")

def main():
    tasks = []
    while True:
        print("\nMenu:")
        print("1. Adicionar Tarefa")
        print("2. Remover Tarefa")
        print("3. Mostrar Tarefas")
        print("4. Sair")
        
        choice = input("Escolha uma opção: ")
        
        if choice == '1':
            task = input("Digite a tarefa: ")
            tasks.append(task)
            print("Tarefa adicionada.")
        elif choice == '2':
            display_tasks(tasks)
            try:
                task_index = int(input("Digite o número da tarefa a ser removida: ")) - 1
                if 0 <= task_index < len(tasks):
                    removed_task = tasks.pop(task_index)
                    print(f"Tarefa '{removed_task}' removida.")
                else:
                    print("Índice inválido.")
            except ValueError:
                print("Por favor, insira um número válido.")
        elif choice == '3':
            display_tasks(tasks)
        elif choice == '4':
            print("Saindo...")
            break
        else:
            print("Opção inválida. Tente novamente.")

if __name__ == "__main__":
    main()